Core/include/Acts/EventData/TrackProxy.hpp

```diff
@@ -618,6 +619,14 @@ class TrackProxy {
     return m_index;
   }
 
+  /// Return the track parameters at the reference surface
+  /// @note The parameters are created on the fly
+  /// @return the track parameters
+  BoundTrackParameters boundParameters() const {
```

Comment:
Should the name clearly indicate that the parameters are at the reference surface? This might not be obvious to the user.